net/libfab/libfab_internal.h

```diff
@@ -276,6 +276,9 @@ struct m0_fab__ep_name {
 
 	/** address in string format as passed by the net layer */
 	char fen_str_addr[LIBFAB_ADDR_STRLEN_MAX];
+
+	/** network address */
+	struct m0_net_ip_addr fen_name;
```

Comment:
Can we replace 'fen_str_addr' with 'fen_name.na_p'?

Reply:
Yes, we will take this task in further improvement, added TODO.